crates/pixi_build_frontend/src/protocols/builders/pixi.rs

```diff
@@ -72,11 +72,13 @@ impl ProtocolBuilder {
             .build_section()
             .map(IsolatedToolSpec::from_build_section);
 
+        let channel_config = ChannelConfig::default_with_root_dir(manifest.path.clone());
```

Comment:
Can we get rid of this and pass in the correct channel config?

Reply:
it is passed later here:
https://github.com/prefix-dev/pixi/pull/2579/files#diff-59e6d7246815b124bae83e48d771a830e31e9b7c57797c8a3ca3ab9979194844R106.

`new` is just responsible to create `another` protocol builder based on the manifest.

but BuildFrontend will overwrite necessary config later